docs/section4/4-11.md

```diff
@@ -0,0 +1,65 @@
+        - Get the baseline system type: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+        - Check if system type is 3 or 4, if it is then continue with indented logic: `if hvac_sys_type_b in ["SYS-3","SYS-3a","SYS-3b, SYS-4"]:`
+            - Get the floor name associated with the zone: `floor_name = match_data_element(B_RMR,Zone,zone_p.id).floor_name`
+            - Get dictionary of zones and hvac systems associated with the floor name: `dict_of_zones_hvac_systems_serving_specific_floor = get_dict_of_zones_hvac_sys_serving_specific_floor(floor_name,B_RMR)`
+            - Create list of hvac system serving the floor from the dictionary: `list_hvac_sys_serving_floor = list(dict_of_zones_hvac_systems_serving_specific_floor.values())`
+            - Loop through and check if any systems are of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `for hvac_flr in list_hvac_sys_serving_floor:`
+                - Check if the system is of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `if baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_flr.id)] in ["SYS-5","SYS-5b","SYS-6, SYS-6b", SYS-7",SYS-7a",SYS-7b",SYS-7c",SYS-8",SYS-8a",SYS-8b",SYS-8c"]:`
+                    - Set hvac_type_check boolean variable to TRUE: `hvac_type_check = TRUE`  
+            - Check if there is system types 5,6,7,8 on the floor via the hvac_type_check boolean variable, if true then check if the proposed zone is served by a multizone HVAC system: `if hvac_type_check == TRUE:`
+                - Check if the zone is served by a multizone system in the proposed by first getting the list of hvac systems serving the zone in the proposed: `list_hvac_systems_p = get_list_hvac_systems_associated_with_zone(P_RMR, zone_p.id)`
+                - Loop through each hvac system and check if any serve multiple zones: `for hvac_pr in list_hvac_systems_p:`
+                    - Check if the hvac system serves more than one zone: `if len(list(dict_hvac_sys_zones_served_p[hvac_pr.id]["Zone_List"].values())) > 1:`
+                        - Loop through terminal units associated with the zone to determine which is/are served by this HVAC system and check if it has reheat: `for terminal_p in zone_p.terminals:`
+                            - Check if the terminal unit is served by the HVAC system: `if terminal_p.served_by_heating_ventilation_air_conditioning_system == hvac_pr.id:`
+                                - Check if it has reheat, if it does then assume its a multizone unit and set the proposed served by multizone boolean variable to TRUE: `if terminal_p.heating_capacity >0:proposed_served_by_multizone = TRUE`   
+                                
+    **Rule Assertion:**
+    - Case 1: For each applicable zone, if no schedule mismatch is observed then pass: `if schedule_mismatch == FALSE: PASS`
```

Comment:
@mkarpman There can be scenarios though where the baseline is multizone and the proposed is single zone and they match. I am thinking of a model we did recently for a school where the baseline was system 5 and then the proposed was all single RTUs. Matching schedules was appropriate in that case so I don't think we would want to fail in that scenario. Maybe if they are matching in the scenarios you describe we can make it undetermined?

Reply:
I don't think it should fail if there is single zone in proposed and MZ in baseline and the schedules match. I have modeled scenarios where this is the case and the schedules should be modeled the same. A recent school comes to mind. 

I think your recommendation below addresses this.

Reply:
Also, what if the single zone is being modeled due to different internal loads rather than different schedules? Then there may not be a schedule mismatch.

Reply:
@mkarpman's reply over email - 	Right. I think modeling the same schedules would also be correct for your #1. The problem is that we cannot definitely establish when #2 applies, and it may apply concurrently with #1, 3 and 4. That said, agree with your scenario above about a school where a zone is served by System 5 in the baseline but a single zone system in the proposed. To accommodate that, I suggest the following change to the logic: PASS should only apply when schedules match AND (baseline & proposed both served by single-zone systems OR baseline & proposed both served by multi-zone systems)